### ISSUE TITLE
[BUG] Download to pdf generates an empty PDF file

### ISSUE THREAD

**Issue by droidtestercan** [AmruthPillai/Reactive-Resume#1004]
**Describe the bug**

Download to pdf generates an empty PDF file

**Product Flavor**
- [X ] Self Hosted

**To Reproduce**

1. After logging in, inputting all of my resume information
2. Click on 'export to PDF'
3. Download the file
4. Click to open it, and blank PDF

**Expected behavior**

I expected to see a pdf generated as what was seen on the app

**Screenshots**

**Desktop (please complete the following information):**

- OS: Ubuntu
- Browser Firefox
- Version 3?

**Comment by Neha-chhajed** [comment]
> **Describe the bug**
> 
> Download to pdf generates an empty PDF file
> 
> **Product Flavor**
> 
> * [X ] Self Hosted
> 
> **To Reproduce**
> 
> 1. After logging in, inputting all of my resume information
> 2. Click on 'export to PDF'
> 3. Download the file
> 4. Click to open it, and blank PDF
> 
> **Expected behavior**
> 
> I expected to see a pdf generated as what was seen on the app
> 
> **Screenshots**
> 
> **Desktop (please complete the following information):**
> 
> * OS: Ubuntu
> * Browser Firefox
> * Version 3?

Hey droidtestercan, Try to use chrome and use Rabbit URL Rewriter extension steps mention in documentation.

**Comment by droidtestercan** [comment]
I'm using docker, and the link to the rabbit url rewriter within the documentation leads to a 404?

**Comment by Neha-chhajed** [comment]
> I'm using docker, and the link to the rabbit url rewriter within the documentation leads to a 404?

Hey use below link -

https://extpose.com/ext/118821/en

**Comment by droidtestercan** [comment]
Why is the extension no longer on the chrome store? If this project is supposed to be secure, why is a dependency not in the chrome store?

**Comment by AmruthPillai** [comment]
Not sure when this extension became delisted, but it couldn't possibly be because it was insecure, but rather that it might go against one of Google's very strict guidelines on extensions modifying requests. It is quite safe, but as of now, you might not even need that rewriter if you are using the right ENV vars on the docker-compose file.

**Comment by droidtestercan** [comment]
@AmruthPillai  Thank you very much for the information. I used the Rabbit rewriter and the pdf came out, I changed the ENV vars on the docker-compose file to be using the ip address of the vm where I have the docker container running. Was there other env variables I should have changed - secret keys were also populated. I appreciate all the work done on this project, and will recommend to others.
